opentok/opentok-meet
# OPENTOK-44484: replaced 'mute on entry' and 'disable mute on entry' buttons with a single toggle button

#### What is this PR doing?
This PR is replacing the 'Mute on Entry' and 'Disable Mute on Entry' buttons with a single 'Toggle Mute on Entry' button that combines their functionality.

#### How should this be manually tested?
- run the app
- toggle mute on entry button
- confirm functionality is correct in a new browser tab

#### What are the relevant tickets?

[OPENTOK-44484](https://jira.vonage.com/browse/OPENTOK-44484)
